settings.gradle.kts

```diff
@@ -222,3 +222,5 @@ include(":system-tests:runtimes:azure-storage-transfer-provider")
 include(":system-tests:runtimes:file-transfer-consumer")
 include(":system-tests:runtimes:file-transfer-provider")
 include(":system-tests:tests")
+include("extensions:data-plane-selector:store:cosmos:data-plane-instance-store-cosmos")
+findProject(":extensions:data-plane-selector:store:cosmos:data-plane-instance-store-cosmos")?.name = "data-plane-instance-store-cosmos"
```

Comment:
i would put this near `data-plane-instance-store-sql` with 
```
include(":extensions:data-plane-selector:store:cosmos:data-plane-instance-store-cosmos")
```
and would remove the `findProject` one